DESCRIPTION

```diff
@@ -9,10 +9,9 @@ Author: Fernando Miguez <femiguez@iastate.edu>,
 Maintainer: Deepak Jaiswal <djaiswal@illinois.edu>,
     David LeBauer <dlebauer@illinois.edu>,
     Fernando Miguez <femiguez@iastate.edu>
+Requires: devtools
```

Comment:
R doesn't parse this field. Probably clearest and most accurate to list it in `Imports` instead.